### DISCUSSION TITLE
Main Screen Buffer in Scout's Bitmap stats

### DISCUSSION THREAD

**Discussion by ghost** [airsdk/Adobe-Runtime-Support#623] [Q&A]
We just noticed that `Main Screen Buffer` in Scout's `Bitmap` stats allocating 6-12Mb RAM (or even more), and curious is it vital or can be cut down to zero (and how), at least when we are not using any display list renderables, only Context3D?

**Comment by ajwfrost** [comment]
Interesting thought .. there is always a framebuffer that's created in memory at start-up, even if you're just using Context3D. But I don't see how this would actually be needed if there was no CPU rendering done. Maybe we can make it into a create-on-first-use sort of buffer and then you'd never use it unless you added something to the display list...

**Comment by ghost** [comment]
Sounds good. Also, in this case I think we would need a way to dispose it at runtime, if we don't need it anymore, because in some rare cases we need to draw text input using DisplayList.
Thanks!
